### PR TITLE
chore(locales): pt-PT

### PR DESCRIPTION
**Pull request by stavares843**
### What this PR does 📖

- Update pt-PT locales, were missing locales

### Which issue(s) this PR fixes 🔨

- Resolve #

### CODE REVIEW

File: common/locales/pt-PT/main.ftl [hunk 1]
```diff
@@ -1,156 +1,276 @@
-uplink = Uplink
-    .home = Início
+uplink = Ligação ascendente
```
Comment (stavares843) [inline common/locales/pt-PT/main.ftl]:
```suggestion
uplink = Uplink
```

File: common/locales/pt-PT/main.ftl [hunk 2]
```diff
@@ -1,156 +1,276 @@
...
+    .home = Casa
```
Comment (stavares843) [inline common/locales/pt-PT/main.ftl]:
```suggestion
    .home = Início
```

File: common/locales/pt-PT/main.ftl [hunk 3]
```diff
@@ -1,156 +1,276 @@
...
     .add = Adicionar
     .call = Chamada
     .coming-soon = Em breve
-    .crash-report = O Uplink falhou. Opcionalmente, guarde o relatório de erro na pasta selecionada.
+    .crash-report = O Uplink sofreu uma avaria. Opcionalmente, guardar o relatório de falha na pasta seleccionada
```
Comment (stavares843) [inline common/locales/pt-PT/main.ftl]:
```suggestion
    .crash-report = O Uplink sofreu uma avaria. Opcionalmente, guardar o relatório de erro na pasta seleccionada
```

File: common/locales/pt-PT/main.ftl [hunk 4]
```diff
@@ -1,156 +1,276 @@
...
     .video-call = Chamada de vídeo
-    .chat = Conversa
-    .chats = Conversas
+    .chat = Bate-papo
```
Comment (stavares843) [inline common/locales/pt-PT/main.ftl]:
```suggestion
    .chat = Conversa
```

File: common/locales/pt-PT/main.ftl [hunk 5]
```diff
@@ -1,156 +1,276 @@
...
+    .chats = Chats
```
Comment (stavares843) [inline common/locales/pt-PT/main.ftl]:
```suggestion
    .chats = Conversas
```

File: common/locales/pt-PT/main.ftl [hunk 6]
```diff
@@ -1,156 +1,276 @@
...
+    .remover = Desfazer amizade
+    .edit-group = Editar grupo
+    .not-creator = Não és o criador
+    .cannot-add-self = Não te podes adicionar a ti próprio, tonto!
     .incoming_requests = Pedidos recebidos
-    .outgoing_requests = Pedidos enviados
+    .outgoing_requests = Pedidos de saída
     .cancel = Cancelar pedido
-    .requested = Recebido
+    .requested = Solicitado
```
Comment (stavares843) [inline common/locales/pt-PT/main.ftl]:
```suggestion
    .requested = Solicitado
```

File: common/locales/pt-PT/main.ftl [hunk 7]
```diff
@@ -1,156 +1,276 @@
...
+    .deny = Negar pedido
     .request-sent = Pedido de amizade enviado!
-    .copied-did = Copiado para a área de transferência!
+    .new_request = Novo pedido de amizade.
+    .copied-did = ID copiado para a área de transferência!
     .unblock = Desbloquear
+    .request-exist = O pedido de amizade já está pendente!
 
-files = Ficheiros
+files = Arquivos
```
Comment (stavares843) [inline common/locales/pt-PT/main.ftl]:
```suggestion
files = Ficheiros
```

File: common/locales/pt-PT/main.ftl [hunk 8]
```diff
@@ -1,156 +1,276 @@
...
-    .no-thumbnail-preview = Não há thumbnail para visualização deste arquivo.
+    .download = Descarregar
+    .uploaded = Carregado
+    .thumbnail-uploaded = Miniatura carregada
+    .no-thumbnail = Sem miniatura
+    .renaming-duplicated = Renomear se estiver duplicado
+    .one-file-to-upload = Ficheiro a carregar
+    .files-to-upload = Ficheiros a carregar
+    .video-thumb = Miniatura de vídeo
+    .image-thumb = Polegar da imagem
```
Comment (stavares843) [inline common/locales/pt-PT/main.ftl]:
```suggestion
    .image-thumb = Miniatura da imagem 
```

File: common/locales/pt-PT/main.ftl [hunk 9]
```diff
@@ -1,156 +1,276 @@
...
+    .download = Descarregar
+    .uploaded = Carregado
+    .thumbnail-uploaded = Miniatura carregada
+    .no-thumbnail = Sem miniatura
+    .renaming-duplicated = Renomear se estiver duplicado
+    .one-file-to-upload = Ficheiro a carregar
+    .files-to-upload = Ficheiros a carregar
+    .video-thumb = Miniatura de vídeo
+    .image-thumb = Polegar da imagem 
+    .doc-thumb = Polegar da primeira página
```
Comment (stavares843) [inline common/locales/pt-PT/main.ftl]:
```suggestion
    .doc-thumb = Previsualização de documento
```

File: common/locales/pt-PT/main.ftl [hunk 10]
```diff
@@ -1,156 +1,276 @@
...
-    .unmute = Remover do Silêncio 
-    .listen = Ativar Áudio
-    .silence = Desativar Áudio
-    .end = Terminar
+remote-controls = Controlos remotos 
+    .in-call = Em chamada
+    .mute = Sem som
+    .unmute = Sem som 
+    .listen = Ouvir
+    .silence = Silêncio
```
Comment (stavares843) [inline common/locales/pt-PT/main.ftl]:
```suggestion
    .silence = Silêncio
```

File: common/locales/pt-PT/main.ftl [hunk 11]
```diff
@@ -1,156 +1,276 @@
...
-    .listen = Ativar Áudio
-    .silence = Desativar Áudio
-    .end = Terminar
+remote-controls = Controlos remotos 
+    .in-call = Em chamada
+    .mute = Sem som
+    .unmute = Sem som 
+    .listen = Ouvir
+    .silence = Silêncio 
+    .end = Fim
```
Comment (stavares843) [inline common/locales/pt-PT/main.ftl]:
```suggestion
    .end = Terminar
```

File: common/locales/pt-PT/main.ftl [hunk 12]
```diff
@@ -3,7 +3,7 @@ uplink = Uplink
     .add = Adicionar
     .call = Chamada
     .coming-soon = Em breve
-    .crash-report = O Uplink falhou. Opcionalmente, guarde o relatório de erro na pasta selecionada.
+    .crash-report = O Uplink sofreu uma avaria. Opcionalmente, guardar o relatório de erro na pasta seleccionada
```
Comment (stavares843) [inline common/locales/pt-PT/main.ftl]:
```suggestion
    .crash-report = O Uplink sofreu um erro. Opcionalmente, guardar o relatório de erro na pasta seleccionada
```

File: common/locales/pt-PT/main.ftl [hunk 13]
```diff
@@ -12,145 +12,265 @@ uplink = Uplink
...
+    .download = Descarregar
+    .uploaded = Carregado
+    .thumbnail-uploaded = Miniatura carregada
+    .no-thumbnail = Sem miniatura
+    .renaming-duplicated = Renomear se estiver duplicado
+    .one-file-to-upload = Ficheiro a carregar
+    .files-to-upload = Ficheiros a carregar
+    .video-thumb = Miniatura de vídeo
+    .image-thumb = Miniatura da imagem 
+    .doc-thumb = Previsualização de documento
```
Comment (stavares843) [inline common/locales/pt-PT/main.ftl]:
```suggestion
    .doc-thumb = Miniatura de pré-visualização de documento
```

File: common/locales/pt-PT/main.ftl [hunk 14]
```diff
@@ -12,145 +12,265 @@ uplink = Uplink
...
-    .no-thumbnail-preview = Não há thumbnail para visualização deste arquivo.
+    .download = Descarregar
+    .uploaded = Carregado
+    .thumbnail-uploaded = Miniatura carregada
+    .no-thumbnail = Sem miniatura
+    .renaming-duplicated = Renomear se estiver duplicado
+    .one-file-to-upload = Ficheiro a carregar
+    .files-to-upload = Ficheiros a carregar
+    .video-thumb = Miniatura de vídeo
+    .image-thumb = Miniatura da imagem
```
Comment (stavares843) [inline common/locales/pt-PT/main.ftl]:
```suggestion
    .image-thumb = Miniatura de pré-visualização
```

File: common/locales/pt-PT/main.ftl [hunk 15]
```diff
@@ -12,145 +12,265 @@ uplink = Uplink
...
-    .renaming-duplicated = Renomeando se Duplicado
-    .one-file-to-upload = Ficheiro para Envio
-    .files-to-upload = Ficheiros para Envio
-    .video-thumb = Thumbnail de vídeo
-    .image-thumb = Thumbnail de imagem
-    .doc-thumb = Thumbnail da primeira página
-    .no-thumbnail-preview = Não há thumbnail para visualização deste arquivo.
+    .download = Descarregar
+    .uploaded = Carregado
+    .thumbnail-uploaded = Miniatura carregada
```
Comment (stavares843) [inline common/locales/pt-PT/main.ftl]:
```suggestion
    .thumbnail-uploaded = Miniatura de pré-visualização carregada
```

File: common/locales/pt-PT/main.ftl [hunk 16]
```diff
@@ -12,145 +12,265 @@ uplink = Uplink
...
+    .popout-player = Leitor de popout 
+    .screenshare = Partilha de ecrã
     
-remote-controls = Controlos Remotos
-    .in-call = Em Chamada
-    .mute = Silenciar
-    .unmute = Remover do Silêncio 
-    .listen = Ativar Áudio
-    .silence = Desativar Áudio
+remote-controls = Controlos remotos
```
Comment (stavares843) [inline common/locales/pt-PT/main.ftl]:
```suggestion
remote-controls = Controlos remotos
```

File: common/locales/pt-PT/main.ftl [hunk 17]
```diff
@@ -12,145 +12,265 @@ uplink = Uplink
...
+    .in-call = Em chamada
+    .mute = Sem som
```
Comment (stavares843) [inline common/locales/pt-PT/main.ftl]:
```suggestion
    .mute = Silenciar
```

File: common/locales/pt-PT/main.ftl [hunk 18]
```diff
@@ -12,145 +12,265 @@ uplink = Uplink
...
+    .unmute = Sem som
```
Comment (stavares843) [inline common/locales/pt-PT/main.ftl]:
```suggestion
    .unmute = Remover do Silêncio
```

File: common/locales/pt-PT/main.ftl [hunk 19]
```diff
@@ -12,145 +12,265 @@ uplink = Uplink
...
-    .mute = Silenciar
-    .unmute = Remover do Silêncio 
-    .listen = Ativar Áudio
-    .silence = Desativar Áudio
+remote-controls = Controlos remotos
+    .in-call = Em chamada
+    .mute = Sem som
+    .unmute = Sem som 
+    .listen = Ouvir
+    .silence = Silêncio
```
Comment (stavares843) [inline common/locales/pt-PT/main.ftl]:
```suggestion
    .silence = Desativar Áudio
```

File: common/locales/pt-PT/main.ftl [hunk 20]
```diff
@@ -12,145 +12,265 @@ uplink = Uplink
...
-    .in-call = Em Chamada
-    .mute = Silenciar
-    .unmute = Remover do Silêncio 
-    .listen = Ativar Áudio
-    .silence = Desativar Áudio
+remote-controls = Controlos remotos
+    .in-call = Em chamada
+    .mute = Sem som
+    .unmute = Sem som 
+    .listen = Ouvir
```
Comment (stavares843) [inline common/locales/pt-PT/main.ftl]:
```suggestion
    .listen = Ativar Áudio
```

File: common/locales/pt-PT/main.ftl [hunk 21]
```diff
@@ -12,145 +12,265 @@ uplink = Uplink
...
+    .not-friends = Já não são amigos
+    .msg-banner = As mensagens são protegidas por encriptação de ponta a ponta e enviadas através de uma rede ponto a ponto.
+    .create-group-chat = Criar um chat de grupo
+    .group-name = Nome do grupo
+    
+favoritos = Favoritos
     .favorites = Favoritos
-    .add = Adicionar aos favoritos
-    .remove = Remover dos favoritos
+    .add = Adicionar aos Favoritos
```
Comment (stavares843) [inline common/locales/pt-PT/main.ftl]:
```suggestion
    .add = Adicionar aos favoritos
```

File: common/locales/pt-PT/main.ftl [hunk 22]
```diff
@@ -12,145 +12,265 @@ uplink = Uplink
...
+    .not-friends = Já não são amigos
+    .msg-banner = As mensagens são protegidas por encriptação de ponta a ponta e enviadas através de uma rede ponto a ponto.
+    .create-group-chat = Criar um chat de grupo
+    .group-name = Nome do grupo
+    
+favoritos = Favoritos
     .favorites = Favoritos
     .add = Adicionar aos favoritos
-    .remove = Remover dos favoritos
+    .remove = Remover dos Favoritos
```
Comment (stavares843) [inline common/locales/pt-PT/main.ftl]:
```suggestion
    .remove = Remover dos favoritos
```

File: common/locales/pt-PT/main.ftl [hunk 23]
```diff
@@ -12,145 +12,265 @@ uplink = Uplink
...
+    .create-group-chat = Criar um chat de grupo
+    .group-name = Nome do grupo
+    
+favoritos = Favoritos
     .favorites = Favoritos
     .add = Adicionar aos favoritos
     .remove = Remover dos favoritos
 
-friends = Amigos
+amigos = Amigos
```
Comment (stavares843) [inline common/locales/pt-PT/main.ftl]:
```suggestion
friends = Amigos
```

**Comment by phillsatellite** [comment]
Found a small issue, it will panic when trying to compile `panic occurred at file common/src/lib.rs, line 22`

Screenshot_2023-04-28_at_10 03 38_AM

File: common/locales/pt-PT/main.ftl [hunk 24]
```diff
@@ -12,145 +12,234 @@ uplink = Uplink
...
+    .profile = Perfil
     .general = Geral 
-    .general-app-language = Idioma da aplicação
-    .general-change-language = Muda o idioma
     .privacy = Privacidade
-    .audio = Áudio
-    .extensions = Extensões
+    .audio = Sons e áudio
+    .ficheiros = Ficheiros
+    .extensões = Extensões
```
Comment (stavares843) [inline common/locales/pt-PT/main.ftl]:
```suggestion
    .extensions = Extensões
```

**Comment by stavares843** [comment]
fixed @phillsatellite 🎉

**Comment by stavares843** [comment]
no more panic 😂

File: common/locales/pt-PT/main.ftl [hunk 25]
```diff
@@ -12,145 +12,265 @@ uplink = Uplink
...
+    .open-folder-description = Abre o diretório local que contém as extensões instaladas.
+    .installed = Instalado
+    .explore = Explorar
+    .settings = Configurações
+
+settings-accessibility = Configurações de acessibilidade
+    .dyslexia = Abrir o Dyslexic
+    .dyslexia-description = O Open Dyslexic pode ajudar alguns utilizadores que sofrem de dislexia, é um tipo de letra personalizado que pode activar.
+
+definições-notificações = Notificações
```
Comment (stavares843) [inline common/locales/pt-PT/main.ftl]:
```suggestion
settings-notifications = Notificações
```

**Pull request merged**